src/components/ListView/ListViewDataSource.js

```diff
@@ -0,0 +1,194 @@
+import invariant from 'fbjs/lib/invariant'
+
+export default class ListViewDataSource {
```

Comment:
Alright cool, I'll do that. It's probably gonna be tomorrow before I have time for this.